### PR TITLE
Adds backend support for pubtator

### PR DESCRIPTION
**Pull request by falquaddoomi**
This PR adds support for the 'pubtator' corpus and removes the 'fulltext' and 'abstracts' corpora.

...I was also about to change the frontend to support the change in corpora, but it looks like it's reading the corpora set from the API's metadata endpoint. Nice work, Vince!

### CODE REVIEW

**Comment by cgreene** [comment]
Just to check - does this change the user-selectable name of the corpus? I think this is a full-text corpus, so I think we should name it something meaningful to users ("PMC Full Text" or "PMCOA Full Text" or the like). I don't think folks will generally be familiar with pubtator.

**Comment by falquaddoomi** [comment]
@cgreene it currently does, yes: users will see "pubtator" in the UI. I'll change the corpora set to a dict of internal IDs to human-readable labels, and modify the frontend to use the label rather than the ID.

**Comment by falquaddoomi** [comment]
@vincerubinetti just FYI, I modified a bit of the frontend code in this PR to deal with the fact that CORPORA_SET is now a dict of (id, label) pairs, not a list of strings. In order to not mess up your code too much, I used just the label part in the same way you were using what was once the corpus ID/label, and modified the backend to take either the corpus ID or the corpus label.

File: server/backend/config.py [hunk 1]
```diff
@@ -38,7 +38,10 @@ def is_truthy(v):
 LOG_LEVEL = os.environ.get("LOG_LEVEL")
 
 # the set of corpora in the data folder
-CORPORA_SET = ("abstracts", "fulltexts")
+CORPORA_SET = {
+    "pubtator": "PMC Full Text",
+    # "preprints"
```
Comment (danich1) [inline server/backend/config.py:43]:
```suggestion
    # "preprints": "BioRxiv/MedRxiv"
```

**Review by danich1 — APPROVED** [review]
LGTM! I suggest one small change, so users will understand the which preprints are being analyzed.

**Pull request merged**